boards/sim/sim/sim/configs/crypto/defconfig

```diff
@@ -68,3 +68,6 @@ CONFIG_START_YEAR=2008
 CONFIG_SYSTEM_DUMPSTACK=y
 CONFIG_SYSTEM_NSH=y
 CONFIG_TESTING_CRYPTO=y
+CONFIG_MBEDTLS_SHA1_ALT=y
```

Comment:
keep the order and add random too

Reply:
@ThePassionate please use "make savedefconfig" to create the defconfig and avoid human error